healme.lic

```diff
@@ -69,32 +73,44 @@ class HealMe
 
   def heal_wound(spells, part, internal)
```

Comment:
❓ What do you think about passing in `spells['Heal Wounds']` here (instead of `spells`)?

Reply:
we can't without eliminating the empath_healing integration from yaml